library/objective-c/EnvoyEngine.h

```diff
@@ -315,6 +315,21 @@ extern const int kEnvoyFilterResumeStatusResumeIteration;
 
 @end
 
+#pragma mark - EnvoyKeyValueStore
+
+@protocol EnvoyKeyValueStore
+
+/// Read a value from the key value store iplementation.
+- (NSString *_Nullable)readValueForKey:(NSString *)key;
+
+/// Remove a value from the key value store implementation.
+- (void)saveValue:(NSString *)value toKey:(NSString *)key;
+
+/// Save a value to the key value store implementation.
```

Comment:
These docstrings are swapped

```suggestion
/// Save a value to the key value store implementation.
- (void)saveValue:(NSString *)value toKey:(NSString *)key;

/// Remove a value from the key value store implementation.
```

Reply:
thanks, good catch